### ISSUE TITLE
There is a bug while adding password to the register form

### ISSUE THREAD

**Issue by SuperexMack** [Luson045/medi-connect#129]
https://github.com/user-attachments/assets/b89d9875-4b0c-464b-bb20-a92817012d29

**Comment by Luson045** [comment]
ohh okay is it for devices with less width? anyway you can work on it @SuperexMack

**Comment by SuperexMack** [comment]
nah this happend while i was checking it on pc but the problem with the small devices the password section is not properly responsive 
![buffff](https://github.com/user-attachments/assets/f548dfe3-8326-40eb-9a58-e75f9bab039e)
i know it is not a big issue but i mentioned it in case i don't have info regarding this

**Comment by Luson045** [comment]
yeah sure you can work on this, i literally didn't knew this @SuperexMack

**Comment by SuperexMack** [comment]
@Luson045  Sure, just give me two days. If I can't resolve it within that time, I'll let you know.

**Comment by SuperexMack** [comment]
@Luson045 I initially worked on the first issue and discovered that it was browser-specific. I was using Microsoft Edge, but after testing it on Chrome, everything worked fine. So, there's no need to fix the first issue. I'll focus on resolving the second one now(that responsive one)

**Comment by Luson045** [comment]
Ohh okay 👍🏻

**Comment by ShreyasNandanwar** [comment]
![Screenshot 2024-10-04 135823](https://github.com/user-attachments/assets/ab1fd6cb-e3ce-4c72-975f-3254d85d65fd)

The password type and the visibility button below conflict with each other.
